Examples/Cocoa/Tests/Objective_CTests/ObjcTests.swift

```diff
@@ -12,24 +12,32 @@ public func == (lhs: JSONDict, rhs: JSONDict) -> Bool {
 
 class ObjcTestSuite: XCTestCase {
 
+    // https://www.ietf.org/rfc/rfc3986.txt
+    let urlRFC3986 = "http://google.com/path/to/greatness?query=parameters&another=one#fragment_life=jump"
+
     func testBasicObjectInitialization() {
+        guard let _ = URLComponents(string: urlRFC3986)?.url else {
+            XCTFail("Could not generate URL using Apple API!")
+            return
+        }
```

Comment:
the point of this change was also the `NSURL` couldn't handle this URL so we should assert that `URL(string: urlRFC3986)` is `nil`

Reply:
per our discussion offline, it seems with toolchain `Version 10.0 (10A255)` NSURL.URLWithString supports RFC3986 ??? This may not be necessary after all